### PR TITLE
docs: clarify gateways don't connect to public internet

### PR DESCRIPTION
**Pull request by jkirschner-hashicorp**
Consul's ingress and terminating gateways are meant to enable connectivity within your organizational network between services outside the Consul service mesh and those within. They are not meant to connect to the public internet.

**See questions I added as Github comments on the diff**

Preview link: https://consul-p22pcdiqu-hashicorp.vercel.app/docs/connect/gateways

### CODE REVIEW

**Comment by jkirschner-hashicorp** [comment]
@blake : could use your eye to confirm the text changes I made to ingress/term gateway descriptions.

File: website/content/docs/connect/gateways/index.mdx [hunk 1]
```diff
@@ -10,8 +10,8 @@ description: >-
 This topic provides an overview of the gateway features shipped with Consul. Gateways provide connectivity into, out of, and between Consul service meshes. You can configure the following types of gateways:
 
 - [Mesh gateways](#mesh-gateways) enable service-to-service traffic between Consul datacenters or between Consul admin partitions. They also enable datacenters to be federated across wide area networks.
-- [Ingress gateways](#ingress-gateways) enable services to accept traffic from outside the Consul service mesh.
-- [Terminating gateways](#terminating-gateways) enable you to route traffic from services in the Consul service mesh to external services.
+- [Ingress gateways](#ingress-gateways) enable services to accept traffic from outside the Consul service mesh that originates from your organizational network.
```
Comment (jkirschner-hashicorp) [inline website/content/docs/connect/gateways/index.mdx:13]: **Question for reviewer**: Should I re-use the same wording that I use later (in the ingress gateway section)?
> Ingress gateways enable connectivity within your organizational network from services outside the Consul service mesh to services within the mesh.

I think that wording might be clearer than what's currently on this line...

File: website/content/docs/connect/gateways/index.mdx [hunk 2]
```diff
@@ -37,8 +37,9 @@ Mesh gateways enable the following scenarios:
 
 -> **1.8.0+:** This feature is available in Consul versions 1.8.0 and newer.
 
-Ingress gateways are an entrypoint for outside traffic. They enable potentially unauthenticated ingress traffic from
-services outside the Consul service mesh to services inside the service mesh.
+Ingress gateways enable connectivity within your organizational network from services outside the Consul service mesh
+to services within the mesh. To accept ingress traffic from the public internet, use Consul's
+[API Gateway](https://www.hashicorp.com/blog/announcing-hashicorp-consul-api-gateway) instead.
```
Comment (jkirschner-hashicorp) [inline website/content/docs/connect/gateways/index.mdx:42]: **Question for reviewer**: Should I even mention the API Gateway at this point? Or leave it out for now?

Comment (karl-cardenas-coding) [reply website/content/docs/connect/gateways/index.mdx:42]: I think it's good to start getting people aware of Consul API Gateway. I vote we keep it.

**Review by karl-cardenas-coding — APPROVED** [review]

File: website/content/docs/connect/gateways/index.mdx (hunk 1 above)
Comment (jkirschner-hashicorp) [reply website/content/docs/connect/gateways/index.mdx:13]: @karl-cardenas-coding : in that case, I'll change this line (and the terminating gateway line) to use the "within your organizational network ____________________________" structure before merging. Thanks!

**Comment by hc-github-team-consul-core** [comment]
:cherries: If backport labels were added before merging, cherry-picking will start automatically.

To retroactively trigger a backport after merging, add backport labels and re-run https://circleci.com/gh/hashicorp/consul/557332.

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit 1a615f63a5115bdbfa66f745ab2e9d817a8a66b7 onto `stable-website` succeeded!

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit 1a615f63a5115bdbfa66f745ab2e9d817a8a66b7 onto `release/1.11.x` succeeded!

**Pull request merged**